src/test/scala/MagicNumbersBlaze.scala

```diff
@@ -0,0 +1,31 @@
+import constants._
```

Comment:
Added MagicNumber* tests to represent the behavior in my local environment.
These tests all tend to pass, while some that titled with "Tends to..." fail sometimes.